config/crd/bases/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -215,27 +215,23 @@ spec:
                                   anyOf:
                                   - type: integer
                                   - type: string
-                                  nullable: true
                                   pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
                                   x-kubernetes-int-or-string: true
                                 description: 'Limits describes the maximum amount
                                   of compute resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/'
-                                nullable: true
                                 type: object
                               requests:
                                 additionalProperties:
                                   anyOf:
                                   - type: integer
                                   - type: string
-                                  nullable: true
                                   pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
                                   x-kubernetes-int-or-string: true
```

Comment:
Becareful removing nullables here. It's removed when `make generate` runs I think.

https://github.com/openshift/oadp-operator/pull/796 might mitigate this issue.

Reply:
@kaovilai Thanks for catching that. I hadn't intended on including this change in the PR. I'll remove it (it's one of the files I have to generally ignore from `make bundle` since it makes changes even when I didn't touch it.